examples/json-type/json_type.md

```diff
@@ -10,7 +10,7 @@ The `json` type can be explained as a union of the simple basic types, `string`,
 
 ## Related links
 - [Access JSON elements](/learn/by-example/access-json-elements/)
-- [Converting from JSON to user defined type with langlib functions](/learn/by-example/converting-from-json-to-user-defined-type-with-langlib-functions/)
-- [Converting from user-defined type to JSON](/learn/by-example/converting-from-user-defined-type-to-json/)
+- [Convert from JSON to user-defined type](/learn/by-example/convert-from-json-to-user-defined-type/)
+- [Convert from user-defined type to JSON](/learn/by-example/converting-from-user-defined-type-to-json/)
```

Comment:
```suggestion
- [Convert from user-defined type to JSON](/learn/by-example/convert-from-user-defined-type-to-json/)
```

Reply:
Don't we need to update the folder names as well?

Reply:
We need to handle it in a separate issue.
It is not related to this bbe link failure.

Reply:
Will this work if we didn't update the folder name?

Reply:
yes. Folder is in the corect name